openmetadata-service/src/main/java/org/openmetadata/service/resources/system/SystemResource.java

```diff
@@ -127,7 +127,7 @@ public Settings getSettingByName(
       @Parameter(description = "Name of the setting", schema = @Schema(type = "string"))
           @PathParam("name")
           String name) {
-    authorizer.authorizeAdmin(securityContext);
+    authorizer.authorizeAdminOrBot(securityContext);
```

Comment:
@harshach @mohityadav766 any reason we had this as admin only and not bot?

Reply:
@TeddyCr these were System settings , like Email , CustomLogo etc, any reason bot should have access to them?

Reply:
don't think it's a good idea to allow bots to view these settings

Reply:
We'll move profiler setting rewtrieval to a specific endpoint